packages/calypso-e2e/src/lib/flows/new-post-flow.ts

```diff
@@ -19,6 +19,7 @@ export class NewPostFlow {
 		this.isLoggedIn = false;
 	}
 
+	// @todo there should be a way to pass a target: SiteType option as it wraps the `EditorPage` below
```

Comment:
I actually don't think we even use this method anymore (I couldn't find any references). I think we can just deprecate this file for now!

Reply:
Ah, cool! 👍🏻